addon/session-stores/cookie.js

```diff
@@ -189,7 +189,7 @@ export default BaseStore.extend({
       document.cookie = `${this._oldCookieName}=`;
     }
 
-    if (isBlank(value)) {
+    if (isBlank(value) && expiration !== 0) {
```

Comment:
I think the point is that in order to clear a cooke you need to write it with an expiration time of 0 or am I missing something?